doc/nestml_language/neurons_in_nestml.rst

```diff
@@ -70,6 +70,69 @@ It is equivalent if either both `inhibitory` and `excitatory` are given, or neit
    * - ``inhibitory``
      - ... should be negative. It is added to the buffer with non-negative magnitude :math:`-w`.
 
+The input ports can also be defined as a vector. For example,
+
+.. code-block:: nestml
+
+   parameters:
+    TWO integer = 2
+   end
+
+   input:
+     foo[TWO] pA <- spike
+     exc_spikes[3] pA <- excitatory spike
+     inh_spikes[3] pA <- inhibitory spike
+   end
+
+In this example, all the spiking input ports are defined as vectors. The integer surrounded by ``[`` and ``]`` determines the size of the vector. The size can either be directly specified as an integer or defined as a parameter of type integer in the ``parameters`` block. The name of the receptors of these ports are denoted by suffixing the ``vector index + 1`` to the port name. In the above example, the receptor name for ``foo[0]`` would be ``FOO_1``.
```

Comment:
I don't understand the ``FOO_1`` notation, is this within the NESTML context? Can we just refer to ``foo[0]`` inside the NESTML model?

Reply:
The `FOO_1` notation is only in the context of NEST where the `receptor_type` is provided in `nest.Connect` calls. It is not in the NESTML context, so I will move it.